Dockerfile.java8

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.11
+MAINTAINER sig-platform@spinnaker.io
+COPY orca-web/build/install/orca /opt/orca
+RUN apk --no-cache add --update bash openjdk8-jre
+RUN adduser -D -S spinnaker
+RUN mkdir plugins && chown -R 100:100 plugins
```

Comment:
Not excited to see these UIDs further propagated, but hopefully this will be cleaned up.

Reply:
Yeah, Brandon is working on it...